src/modules/addcomment.js

```diff
@@ -0,0 +1,15 @@
+const addcomment = (dataid, name, text) => {
+  fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/cXyK0mK0Ha1BkQqOi8Kz/comments',
+    {
+      method: 'post',
+      body: JSON.stringify({
+        item_id: dataid,
+        username: name,
+        comment: text,
+      }),
+      headers: { 'content-type': 'application/json' },
+    })
+    .then((resc) => resc.text());
```

Comment:
- [ ] As stated 😃 by the previous reviewer kindly 😊 try to implement async and await with the fetch method to send and receive data from the API. The async and await keywords enable asynchronous, promise-based behavior to be written in a cleaner style.💯